### DISCUSSION TITLE
[Issue] Search product issue

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#488] [Ideas]
Hi,
1-when I clicked on search button without entering any word ,search result give me some products.
2-search autocomplete didn't work, that is make search boring.
3-search result should be autocomplete and gives unlimited results  with scroll bar.
4-All new searching method provide multiple search criteria: barcode/SKU/Name (Autocomplete all of them)
thank you
**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment:**
 - NexoPOS 4.x vesrion: [e.g. 4.5.2]
 - Premium Extensions: [e.g Nexo Print Adapter (4.5.0), Gastro (4.5.2)]

**Comment by Blair2004** [comment]
You need to fill all the template for providing more information about your setup. If you're suggesting a new feature, this is discussed on the "Discussion" Tab.

1 - How this is a problem ?
2 - For searching the "Search" button should be pressed, but yes this can be improved.
3 - No unlimited result. We need to restrict the result for better performances. The user search should be more accurate.
4 - Search can be made using SKU, Name and Barcode.

**Reply by boudo2017** [reply]
when I clicked on search button without entering any word ,search result give me some products.
It is an issue
